### PR TITLE
styling

### PR DESCRIPTION
**Pull request by Moukarnold**

### CODE REVIEW

File: components/dictaphone/Dictaphone.js [hunk 1]
```diff
@@ -6,9 +6,12 @@ import {
   Box,
   Box1,
   ContainerMain,
+  Mic,
   Navbar1,
   ScreenBox,
+  Microphone,
   ScreenBox1,
+  Boutons,
```
Comment (doemser) [inline components/dictaphone/Dictaphone.js]: Boutons? This is a single "Button"

File: components/dictaphone/Dictaphone.js [hunk 2]
```diff
@@ -6,9 +6,12 @@ import {
   Box,
   Box1,
   ContainerMain,
+  Mic,
   Navbar1,
   ScreenBox,
+  Microphone,
   ScreenBox1,
```
Comment (doemser) [inline components/dictaphone/Dictaphone.js:9]: Screenbox1 is an awful naming, are you also sure about it being a div?

File: components/dictaphone/Dictaphone.js [hunk 3]
```diff
@@ -43,27 +46,38 @@ const Dictaphone = ({ language }) => {
...
-          <p>Microphone: {listening ? "on" : "off"}</p>
+          <Microphone>
+            <p>
+              <h5>🎙️: {listening ? "On" : "off"}</h5>
+            </p>{" "}
+          </Microphone>
+          <Box1>
+            <div>
+              <p>{transcript}</p>
+            </div>
```
Comment (doemser) [inline components/dictaphone/Dictaphone.js:57]: do you need this div here?

File: components/dictaphone/Dictaphone.js [hunk 4]
```diff
@@ -43,27 +46,38 @@ const Dictaphone = ({ language }) => {
...
+    <ScreenBox1>
+      {browserSupportsSpeechRecognition && (
         <div>
-          <p>Microphone: {listening ? "on" : "off"}</p>
+          <Microphone>
+            <p>
+              <h5>🎙️: {listening ? "On" : "off"}</h5>
+            </p>{" "}
+          </Microphone>
+          <Box1>
```
Comment (doemser) [inline components/dictaphone/Dictaphone.js:54]: Box1 is an awful naming.

File: components/dictaphone/Dictaphone.js [hunk 5]
```diff
@@ -43,27 +46,38 @@ const Dictaphone = ({ language }) => {
...
+            <div>
+              <p>{transcript}</p>
+            </div>
+          </Box1>
+          <Mic></Mic>
```
Comment (doemser) [inline components/dictaphone/Dictaphone.js]: Can this be a self-closing element?

File: components/styledComponents/Container.styled.js [hunk 6]
```diff
@@ -54,45 +54,57 @@ export const Navbar1 = styled.div`
   border-radius: 10px;
   border: solid 2px;
   background-color: #f8ecd0;
+  padding: 1px;
   display: flex;
-  justify-content: center;
+  flex-direction: row;
   justify-content: space-between;
-  padding: 1px;
+  //margin-bottom: 40px;
```
Comment (doemser) [inline components/styledComponents/Container.styled.js]: remove unused code

**Review by doemser — CHANGES_REQUESTED** [review]

File: components/dictaphone/Dictaphone.js (hunk 1 above)
Comment (Moukarnold) [reply components/dictaphone/Dictaphone.js]: it is a styled "button". i should remove the "s" at he end

File: components/dictaphone/Dictaphone.js (hunk 2 above)
Comment (Moukarnold) [reply components/dictaphone/Dictaphone.js:9]: yes the component dictaphone comes with his display feature on the page screening to be manipulated

File: components/dictaphone/Dictaphone.js (hunk 3 above)
Comment (Moukarnold) [reply components/dictaphone/Dictaphone.js:57]: ja it is for the manipulation and positioning

File: components/dictaphone/Dictaphone.js (hunk 5 above)
Comment (Moukarnold) [reply components/dictaphone/Dictaphone.js]: yes

File: components/dictaphone/Dictaphone.js (hunk 4 above)
Comment (dimitriosxmi) [reply components/dictaphone/Dictaphone.js:54]: Agreed, but who I am to say no if the gentleman wants to have 1 there :)

File: components/difficulty/Difficulty.js [hunk 7]
```diff
@@ -15,9 +15,9 @@ export default function Difficulty(props) {
         onChange={handleDifficulty}
       >
         <option value="">--Please choose an option--</option>
-        <option value="Basic">Basic</option>
-        <option value="Intermediate">Intermediate</option>
-        <option value="Advanced"> 🥋 Advanced</option>
+        <option value="Basic">🧍🏼‍♂️ Basic</option>
+        <option value="Intermediate"> 🚶🏼‍♂️Intermediate</option>
+        <option value="Advanced"> 🏃🏼‍♂️ Advanced</option>
```
Comment (dimitriosxmi) [inline components/difficulty/Difficulty.js:20]: Note: Nice emojicon utilization here :D I dig that!

**Review by dimitriosxmi — APPROVED** [review]
## LGTM Overall
Note: noticed various small nicnacs here and there but I prefer the idea that everyone has their own coding style.

**Pull request merged**